packages/providers/upload-aws-s3/lib/index.js

```diff
@@ -83,8 +83,7 @@ module.exports = {
        */
       getSignedUrl(file, customParams = {}) {
```

Comment:
Looks like the customParams of this function are not used

Reply:
true! :) removed it